src/grandorgue/sound/scheduler/GOSoundGroupWorkItem.h

```diff
@@ -25,8 +27,8 @@ class GOSoundGroupWorkItem : public GOSoundWorkItem, public GOSoundBufferItem {
   GOMutex m_Mutex;
   GOCondition m_Condition;
   unsigned m_ActiveCount;
-  unsigned m_Done;
-  volatile bool m_Stop;
+  std::atomic_uint m_Done;
+  std::atomic_bool m_Stop;
 
```

Comment:
Because you have changed the types to atomic, you also have to add `load` and `store` everywhere with these variables in  src/grandorgue/sound/scheduler/GOSoundGroupWorkItem.cpp

Reply:
Accessing an atomic variable will automatically result in atomic loads and stores:

- Normal read access: https://en.cppreference.com/w/cpp/atomic/atomic/operator_T
- Normal write access: https://en.cppreference.com/w/cpp/atomic/atomic/operator%3D

I can add it nevertheless, if you consider it useful for readability.

Reply:
> I can add it nevertheless, if you consider it useful for readability.

Yes, it would be super. Readability is an important issue.